packages/block-editor/src/components/global-styles/use-global-styles-output.js

```diff
@@ -698,7 +698,7 @@ export const getNodesWithSettings = ( tree, blockSelectors ) => {
 		( [ blockName, node ] ) => {
 			const blockPresets = pickPresets( node );
 			const blockCustom = node.custom;
-			if ( ! isEmpty( blockPresets ) || blockCustom ) {
+			if ( Object.keys( blockPresets ).length > 0 || blockCustom ) {
```

Comment:
`blockPresets` will always be an object:

https://github.com/WordPress/gutenberg/blob/4a6972b681196432663b9f4c5ce53efb8ba80b49/packages/block-editor/src/components/global-styles/use-global-styles-output.js#L675